ya_httpx_client/provider_auto_balance.py

```diff
@@ -1,10 +1,10 @@
 from typing import TYPE_CHECKING
 from datetime import datetime
 if TYPE_CHECKING:
-    from ya_httpx_client.session import Cluster
+    from ya_httpx_client.cluster import Cluster
 
 
-class SimpleLoadBalancer:
+class SimpleLoadBalancer:  # pylint: disable=too-few-public-methods
```

Comment:
Again, a global disable is fine here so either add `too-few-public-methods` to the disable list in `.pylintrc` or put in this:
```
[DESIGN]
min-public-methods=0
```